conjure-java-core/src/integrationInput/java/com/palantir/product/SimpleEnum.java

```diff
@@ -21,6 +24,9 @@
 public final class SimpleEnum {
     public static final SimpleEnum VALUE = new SimpleEnum(Value.VALUE, "VALUE");
 
+    public static final List<SimpleEnum> VALUES =
```

Comment:
do we need to defend against an enum value named `VALUES`?

Reply:
Oof good point